parser/testdata/applicationrequestfunctions.json

```diff
@@ -1,8 +1,6 @@
-{
-    "functions": [
-      {
-        "name": "sendRejectionEmailFunction",
-        "operation": "http://myapis.org/application.json#emailRejection"
-      }
-    ]
-  }
\ No newline at end of file
+[
+  {
```

Comment:
why we remove the outer `function` properties？in spec 0.8，the reference resource should be an object (not an array), this is describe in:

- https://github.com/serverlessworkflow/specification/blob/0.8.x/specification.md#workflow-definition-structure

Reply:
The spec have this definition

> The functions property can be either an in-line [function](https://github.com/serverlessworkflow/specification/blob/0.8.x/specification.md#Function-Definition) definition array, or an URI reference to a resource containing an array of [functions](https://github.com/serverlessworkflow/specification/blob/0.8.x/specification.md#Function-Definition) definition. Referenced resource can be used by multiple workflow definitions.

The [sdk-net](https://github.com/serverlessworkflow/sdk-net/blob/main/src/ServerlessWorkflow.Sdk.UnitTests/Resources/functions/petstore.json) also follow that definition

Reply:
If we are going to update it, I suggest drop it from this PR and open a new issue to address/discuss it.

Reply:
Add the ability to unmarshal both cases object or array. What do you think?

Also, opened the issue #168

Reply:
Back the external resource as object.